### PR TITLE
Styled tables branch

### PR DESCRIPTION
**Pull request by JosephDClark97**
Created all of the styled.divs for main page table, as well as set up the main page table.

### CODE REVIEW

File: src/components/CoinTable/CoinTable.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { React } from "react";
+// import { Chart } as ChartJS;
```
Comment (yoloOnTheBattlefield) [inline src/components/CoinTable/CoinTable.js:2]: is this code needed?

File: src/components/CoinTable/CoinTable.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import "./styles.css";
+
+class CoinTable extends React.Component {
+    state = {
+      list: [],
+      value: "",
+    };
+
+    componentDidMount() {
+      fetch('https://api.coingecko.com/api/v3')
```
Comment (yoloOnTheBattlefield) [inline src/components/CoinTable/CoinTable.js:13]: you can use async await

File: src/components/CoinTable/CoinTable.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        .then(result => {
+          this.setState({result: [...result.content]})
```
Comment (yoloOnTheBattlefield) [inline src/components/CoinTable/CoinTable.js:15]: format this properly
this.setState({ result: [...result.content] })

File: src/components/CoinTable/CoinTable.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        })
+      }
+    
+      render() {
+        return ( <div> {this.state.list.map(el => <div>{el.name}</div>)} </div>)
+      }
```
Comment (yoloOnTheBattlefield) [inline src/components/CoinTable/CoinTable.js:21]: here as well

File: src/components/CoinTable/CoinTable.styles.js [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+  color:#fff;
+`;
+
+
+export const TwentyFourHourChangeContainer = styled.div`
+text-align:center;
+font-family:Poppins;
+font-size:18px;
+font-weight:600;
+color:#fff;
```
Comment (yoloOnTheBattlefield) [inline src/components/CoinTable/CoinTable.styles.js:100]: indent

File: src/components/ThemeChangeBtn/ThemeChangeBtn.styles.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+`;
+
+export const ThemeButton = styled.button`
+display:flex;
+justify-content:center;
+width: 67px;
+height: 63px;
+background-color:#290132;
+cursor:pointer;
+border-radius:10px;
```
Comment (yoloOnTheBattlefield) [inline src/components/ThemeChangeBtn/ThemeChangeBtn.styles.js:15]: indent

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/components/CoinTable/CoinTable.js (hunk 1 above)
Comment (JosephDClark97) [reply src/components/CoinTable/CoinTable.js:2]: probably not the Chart and Chart JS

Comment (JosephDClark97) [reply src/components/CoinTable/CoinTable.js:2]: will likely need that for the actual two other charts, right ?

File: src/components/CoinTable/CoinTable.js (hunk 2 above)
Comment (JosephDClark97) [reply src/components/CoinTable/CoinTable.js:13]: async componentDidMount() {
     await fetch('https://api.coingecko.com/api/v3')
        .then(result => {
          this.setState({result: [...result.content]})
        })
      }

File: src/components/CoinTable/CoinTable.js (hunk 3 above)
Comment (JosephDClark97) [reply src/components/CoinTable/CoinTable.js:15]: is this what you mean? 
         this.setState(
            {
              result: [...result.content]
            }
            )
        })
      }

File: src/components/CoinTable/CoinTable.js (hunk 4 above)
Comment (JosephDClark97) [reply src/components/CoinTable/CoinTable.js:21]: not sure what u mean         
return ( 
        <div> 
          {
          this.state.list.map
          (
            el => 
            <div>
            {el.name}
            </div>
          )
          } 
          </div>
          )

File: src/components/CoinTable/CoinTable.styles.js (hunk 5 above)
Comment (JosephDClark97) [reply src/components/CoinTable/CoinTable.styles.js:100]: done, indented all the rest

File: src/components/ThemeChangeBtn/ThemeChangeBtn.styles.js (hunk 6 above)
Comment (JosephDClark97) [reply src/components/ThemeChangeBtn/ThemeChangeBtn.styles.js:15]: done

**Pull request merged**